app/src/modules/settings/routes/data-model/fields/components/field-select.vue

```diff
@@ -199,8 +215,14 @@ export default defineComponent({
 
 		const nestedFields = computed(() => props.fields.filter((field) => field.meta?.group === props.field.field));
 
+		// disable editing the autogenerated item field.
+		const isDisabled = computed(
+			() => props.field.field === 'item' && props.field.meta?.hidden === true && localType.value === 'm2o'
+		);
```

Comment:
This would mean that any hidden field that is called `item`, that is hidden and that is inside a m2o relation will be disabled, right? I'm wondering if there is a way to specifically detect the auto-generated `ìtem` field.

Reply:
the field itself is considered an m2o field, hidden and called `item`, there isn't another good way to detect auto-generated currently, I spent some time going down that road before settling on this method.